test/agentchat/contrib/vectordb/test_mongodb.py

```diff
@@ -0,0 +1,288 @@
+import logging
+import os
+from time import sleep
+from typing import List
+
+import pytest
+
+from autogen.agentchat.contrib.vectordb.base import Document
+
+try:
+    import pymongo
+    import sentence_transformers
+
+    from autogen.agentchat.contrib.vectordb.mongodb import MongoDBAtlasVectorDB
+except ImportError:
+    # To display warning in pyproject.toml [tool.pytest.ini_options] set log_cli = true
+    logger = logging.getLogger(__name__)
+    logger.warning(f"skipping {__name__}. It requires one to pip install pymongo or the extra [retrievechat-mongodb]")
+    pytest.skip(allow_module_level=True)
+
+from pymongo.collection import Collection
+from pymongo.errors import OperationFailure
+
+logger = logging.getLogger(__name__)
+
+MONGODB_URI = os.environ.get("MONGODB_URI")
```

Comment:
A default value should be given.

Reply:
One has to have an Atlas Cluster set up for this to run.